index.html

```diff
@@ -49,56 +49,56 @@ <h2>Skills</h2>
       </section>
       </div>
 
-      <section id="bio">
+      <section class="details__bio">
         <h2>Biografie</h2>
-        <p>
+        <p class="details__bio--f-10">
             Im Anschluss an mein Marketing- & Medienmanagment Studium bin ich in verschiedenen Agenturen tief in die Welt des On- und Offline Marketings eingetaucht. Thematisch habe ich in den letzten Jahren namhafte Kunden in den Bereichen Medizin, Fashion und Tourismus betreut. 
             <br>  
             <br> 
             Auf der Suche nach neuen beruflichen Herausforderungen und Perspektiven bin ich im Herbst 2020 auf <a href="https://www.neuefische.de" alt="" rel="noopener noreferrer" target="_blank"> neue fische </a> aufmerksam geworden. Mit Programmierung kam ich - bei E-Mail Newslettern, bei der Betreuung von Kunden Websites etc. - während meiner gesamten Laufbahn immer wieder in Kontakt. In der beruflichen Position zur Steuerung der Projekte und strategischer Beratung war jedoch, trotz großem Interesse, nie die Zeit sich tiefergehend mit der Materie zu beschäftigen, geschweige denn die Fähigkeiten zu lernen. Doch das ist nun anders! Ich nehme mir die Zeit mich in einem 3-monatigen Bootcamp zum Web Developer ausbilden zu lassen.<br> <br> <strong>Mein Ziel? - Marketing & Web Development verknüpfen, um die Synergieeffekte mit größtmöglichem Output zu nutzen!</strong>         </p>
       </section>
 
-      <section id="social">
+      <section class="main__social">
```

Comment:
Again, new BEM block:
```suggestion
      <section class="social">
```